graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,359 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import ot
+import numpy as np
+from sklearn.utils import check_array
+
+from .base import BaseAlign
+from .sign_flips import SignFlips
+from .orthogonal_procrustes import OrthogonalProcrustes
+
+
+class SeedlessProcrustes(BaseAlign):
+    """
+    Matches two datasets using an orthogonal matrix. Unlike
+    :class:`~graspy.align.OrthogonalProcrustes`, this does not requires a
+    matching between entries. It can even be used in the settings when the two
+    datasets do not have the same number of entries.
+
+    In graph setting, it is used to align the embeddings of two different
+    graphs, when it requires some simultaneous inference task and no 1-1
+    matching between the vertices of the two graphs can be established, for
+    example, inside of the test for the equivalence of the latent distributions
+    (see: :class:`~graspy.inference.LatentDistributionTest`).
```

Comment:
```suggestion
```

Reply:
again, can ignore this one if you feel strongly, but I think removing this would make more concise and this isn't necessary here. could also go in the notes or more importantly any tutorials

Reply:
should undoubtably be in some tutorial eventually. idk. i wouldn't say i feel "strongly" per se, but this feels to serve a purpose, because we are graph package after all, and this is the complement of the statement made in the orthogonal procrustes.

Reply:
so i'd vote to leave this in.